src/ui/components/Toolbar.tsx

```diff
@@ -170,8 +355,8 @@ function ToolbarButton({
           className="absolute h-2 w-2 rounded-full bg-secondaryAccent"
           style={{
             // FE-1096 Aiming for pixel perfect badge alignment over icons with inconsistent shapes
-            right: name === "comments" ? ".9rem" : "1em",
-            top: name === "debugger" ? ".6em" : "0.5em",
+            right: name === "comments" ? ".7rem" : ".8em",
+            top: name === "debugger" ? ".5em" : "0.5em",
```

Comment:
This seems unnecessary, given the SVGs are all the same size.

FWIW they _don't_ line up:
<img width="118" alt="Screen Shot 2023-06-28 at 12 07 29 PM" src="https://github.com/replayio/devtools/assets/29597/83d79cfd-de50-4229-b98a-a5d4732e9cc2">

Seems like we should just use flex styles to center-align the whole column?

Reply:
>Seems like we should just use flex styles to center-align the whole column?

They are center-aligned, that code positions to the little notification dots, which do need to be different for those two icons because their outlines/shapes are different.

Reply:
I see.